resource/systemd/unit/unit.go

```diff
@@ -0,0 +1,543 @@
+// Copyright © 2016 Asteris, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package unit
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/asteris-llc/converge/resource"
+	"github.com/asteris-llc/converge/resource/systemd"
+	"github.com/coreos/go-systemd/dbus"
+)
+
+// Unit represents a systemd units
+// TODO Enable Parallelization of this Unit
+type Unit struct {
+	// The name of this unit as in "foo.service"
+	Name string // (either just file names or full absolute paths if the unit files are residing outside the usual unit search paths
+
+	// Active determines whether this unit should be active or inactive
+	Active bool
+
+	/*
+		UnitFileState encodes the install state of the unit file of FragmentPath.
```

Comment:
why is this indented like this? Also, you don't need to repeat the doc block here, Preparer is sufficient for fields you've already documented.